lib/index.ts

```diff
@@ -95,12 +95,12 @@ export function concatRegex(...regexes: (string | RegExp)[]) {
 
 export function functionRegex(
   funcName: string | null,
-  paramList?: string[],
+  paramList?: string[] | null,
```

Comment:
I take it the typedef is `string[] | undefined | null` just so you can call `functionRegex('name', null, options)` instead of `functionRegex('name', undefined, options)`?

Reply:
Yep. Just a preference, but passing undefined doesn't look good.